programmers/난이도별/level01.제일_작은_수_제거하기/SSSOy.java

```diff
@@ -0,0 +1,26 @@
+import java.util.*;
+
+class Solution {
+    public int[] solution(int[] arr) {
+        
```

Comment:
if (arr.length < 2) return new int[]{-1};
```suggestion
    public int[] solution(int[] arr) {
        if (arr.length < 2) return new int[]{-1};
```
최상단에 이렇게 바로 예외처리를 하면 좋을 것 같아요!